examples/browser-to-server/index.js

```diff
@@ -0,0 +1,39 @@
+import { createLibp2p } from 'libp2p'
+import { Noise } from '@chainsafe/libp2p-noise'
+import { multiaddr } from '@multiformats/multiaddr'
+import first from "it-first";
+import { pipe } from "it-pipe";
+import { fromString, toString } from "uint8arrays";
+import { webRTC } from 'js-libp2p-webrtc'
+
+  let stream;
```

Comment:
Is this indention on purpose? o.O

Reply:
Or did you forget to define `(async () => {` on top?

Reply:
Even better, probably only want to do it once you know the DOM is initialised/ready?